packages/react/src/components/InAppMessaging/hooks/useMessageProps/useMessageProps.ts

```diff
@@ -33,28 +32,16 @@ export default function useMessageProps(
     }
   }, [onDisplay, shouldRenderMessage]);
 
-  const hasPrimaryButton = !isEmpty(primaryButton);
-  const hasSecondaryButton = !isEmpty(secondaryButton);
-  const hasButtons = hasPrimaryButton || hasSecondaryButton;
```

Comment:
Moved these to `MessageLayout`. These were added previous to the addition of `MessageLayout` to repeating the same logic in each message component.

Reply:
Thanks for making life easier for future maintainers!